### ISSUE TITLE
[Improvement] <Add projects description, problems and expected outcome in README documentation>

### ISSUE THREAD

**Issue by FrancoFrancis** [creativecommons/search#161]
## Problem
There is only documentation about setting up the project in the Projects README, although that may not neccessarily be a problem.

## Description
By adding a detailed description of the project, the problems associated with the project and the expected outcome just as on the outreachy website I believe the project will be more complete. if this issue is approved Would like to work with a mentor to add additional description if needed.

## Additional context
Implementing this would make it that just by reading the README documentation you would be able to understsand the details and complete direction of the project.

## Implementation

- [x] I would be interested in implementing this feature.

**Comment by FrancoFrancis** [comment]
hi @sara kindly check if this issue is to be worked on. thanks :)

**Comment by possumbilities** [comment]
@FrancoFrancis I'd be interested to see what suggestions anyone would have here on what should be appended to the `README.md` file to improve things.

**Comment by FrancoFrancis** [comment]
> 



> @FrancoFrancis I'd be interested to see what suggestions anyone would have here on what should be appended to the `README.md` file to improve things.

okay, Can I go on to the community to start an open discussion ?

**Comment by possumbilities** [comment]
@FrancoFrancis You can direct them here to have the discussion so the record is on this Issue, but sure, please do! ❤️

**Comment by FrancoFrancis** [comment]
> @FrancoFrancis You can direct them here to have the discussion so the record is on this Issue, but sure, please do! ❤️

okay on it.

**Comment by dikehprosper** [comment]
@possumbilities  @FrancoFrancis.... I think the most important part is a documentation that talks about the functionality and what the project does in the README

**Comment by mbonamensa** [comment]
@FrancoFrancis correct me if I'm wrong. Do you mean the readme of the project should include problems that are expected to be resolved within the project? For example "We're looking to refactor the code base, etc" Something like that?

**Comment by FrancoFrancis** [comment]
> @FrancoFrancis correct me if I'm wrong. Do you mean the readme of the project should include problems that are expected to be resolved within the project? For example "We're looking to refactor the code base, etc" Something like that?

Yes.... the issue is open for discussion on what should be appended to the readme to make it as informative as possible this is to make sure getting started as a user or as a developer/contributor is as easy as possible and also to reduce noice, because it is the one of the most important parts of any Github project.

**Comment by FrancoFrancis** [comment]
Hi @possumbilities can I proceed to working on this issue while its still open for discursion?
